docs/adding_packages/dependencies.md

```diff
@@ -182,10 +182,12 @@ Currently, these are:
 * OpenSSL: `[>=1.1 <4]` for libraries known to be compatible with OpenSSL 1.x and 3.x
 * CMake: `[>3.XX <4]`, where `3.XX` is the minimum version of CMake required by the relevant build scripts
 * Libcurl: `[>=X.YY <9]`, where `X.YY` is the minimum version of Libcurl required, starting from `7.78`
+* Zlib: `[>=1.2.11 <2]`
+* Libpng: `[>=1.X <2]`, where `1.X` is the minimum version of libpng required by the recipe
```

Comment:
I'd recommend `[>=1.6 < 6]`, unless a higher one is needed - I know it's unlikely we'll ever solve to 1.5, but those are too old.

Reply:
I agree, keeping a real version range is much safer for users when reading the documentation.